### ISSUE TITLE
[bitnami/wordpress] Persist .htaccess

### ISSUE THREAD

**Issue by v-six** [bitnami/charts#2294]
**Which chart**: 
bitnami/wordpress

**Is your feature request related to a problem? Please describe.**
I am managing an infrastructure for a lot of wordpress. Your charts are almost perfect ! Since I would like to let users manage their own wordpress, they need a `.htaccess` file. 

The problem is when the container reboot, `.htaccess` file disappear. Is there a way to persist it ?

**Describe the solution you'd like**
I know that all persisted data are in `/bitnami` and I tried to put in it my `.htaccess` file. 
My suggestion is to restore the `.htaccess` to `/opt/bitnami/wordpress/` if it exists in `/bitnami/wordpress/` folder.

**Describe alternatives you've considered**
I tried to store my `.htaccess` file into `/bitnami/wordpress/` and made a symlink to make it available into `/opt/bitnami/wordpress/` but the problem is the same, my symlink is deleted on container reboot.

**Additional context**

**Comment by dani8art** [comment]
Hi @v-six Could it fit your requirements? https://github.com/bitnami/charts/tree/master/bitnami/wordpress#disabling-htaccess

Also, you can try this

- modify the image

```
FROM bitnami/wordpress

USER root

RUN mkdir -p /bitnami/wordpress/ && \
  touch /bitnami/wordpress/.htaccess && \
  ln -s /bitnami/wordpress/.htaccess /opt/bitnami/wordpress/.htaccess

USER 1001
```

- Use your own image in the chart

```
helm install wc \
  --set image.registry=docker.io \
  --set image.repository=darteaga/wordpress \
  --set image.tag=with-htaccess \
  bitnami/wordpress
```

**Comment by v-six** [comment]
Hi @dani8art,

Thanks for your answer.

- CM solution :x: : 
https://github.com/bitnami/charts/tree/master/bitnami/wordpress#disabling-htaccess don't fit my requirments. My problem is that my users will add some rules in this file (via some wordpress plugins), so if I mount a config map, all the content added by users will be deleted on reboot.

- Custom image :
It works very well with a custom docker image, but I was expecting a more "official" solution as I think this is a common wordpress practice to work with `.htaccess` file.

- Post init script :x: :
I tried earlier to add a script in `/docker-entrypoint-init.d` in order to add my `.htaccess` (https://github.com/bitnami/bitnami-docker-wordpress/blob/master/5/debian-10/rootfs/post-init.sh#L5). I was hoping for it to run every time the container starts, but it's not, thoose scripts run only after wordpress installation.

I really understand performance and security reasons why the `.htaccess` file is disabled by default. But as it's a very common practice in wordpress, I think it should be supported or at least possible to work with a persisted `.htaccess` file.

**Comment by dani8art** [comment]
Hi @v-six that sounds good, we are going to open an internal task in order to evaluate and handle it. Unfortunately, we can not give you an ETA but we will keep you updated. Thank you so much for your feedback.

**Comment by dani8art** [comment]
Hi @v-six, this feature should be included in `bitnami/wordpress@9.2.0` please upgrade your repo and try it. 

```bash
$ helm repo update
$ helm search repo bitnami/wordpress
# NAME                    CHART VERSION   APP VERSION     DESCRIPTION                                       
# bitnami/wordpress       9.2.0           5.4.0           Web publishing platform for building blogs and ...
```
Thank you for your feeback

**Comment by v-six** [comment]
Hi @dani8art,

This new feature works like a charm ! `.htaccess` file is now persisted, and a pod reboot / crash does not affect this file anymore.

Many thanks 👍
